src/components/checkbox/checkbox.scss

```diff
@@ -218,8 +218,10 @@ $_md-checkbox-indeterminate-checked-easing-function: cubic-bezier(0.14, 0, 0, 1)
 }
 
 md-checkbox {
-  &, label {
-    cursor: pointer;
+  cursor: pointer;
+
+  label {
+    cursor: inherit;
```

Comment:
Can you un-nest the rule out from md-checkbox and just use md-checkbox-layout? This will keep the specificity down.

Add a comment explaining the intent of adding cursor: inherit.